examples/lambda-functions/get-by-id.ts

```diff
@@ -30,6 +30,14 @@ const tableName = process.env.SAMPLE_TABLE;
 
 class Lambda implements LambdaInterface {
 
+  @tracer.captureMethod()
+  public async getUuid(): Promise<string> {
+    // Request a sample random uuid from a webservice
+    const res = await got('https://httpbin.org/uuid');
+    
+    return JSON.parse(res.body).uuid;
+  }
+
   @tracer.captureLambdaHandler()
   @logger.injectLambdaContext({ logEvent: true })
   @metrics.logMetrics()
```

Comment:
Change to `@metrics.logMetrics({ throwOnEmptyMetrics: true, captureColdStartMetric: true })`